<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1162</width>
    <height>842</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_title">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>0</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_new">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>4</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="whatsThis">
      <string>save it</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_save">
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>4</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="whatsThis">
      <string>save it</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_start">
     <property name="geometry">
      <rect>
       <x>82</x>
       <y>4</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="whatsThis">
      <string>save it</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_export">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>4</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="whatsThis">
      <string>save it</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>45</y>
      <width>230</width>
      <height>741</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>4</y>
       <width>231</width>
       <height>731</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="indentation">
      <number>10</number>
     </property>
     <attribute name="headerVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="headerCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>检查工程</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>47</y>
      <width>921</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>530</y>
      <width>921</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1162</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(F)</string>
    </property>
    <addaction name="actionNew"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>视图(V)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_W">
    <property name="title">
     <string>窗口(W)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(H)</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_V"/>
   <addaction name="menu_W"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
